BuildHelpers/Scripts/Set-BuildVariable.ps1

```diff
@@ -111,11 +113,15 @@ $BuildHelpersVariables = @{
     ProjectPath = ${Build.Vars}.ProjectPath
     BranchName  = ${Build.Vars}.BranchName
     CommitMessage = ${Build.Vars}.CommitMessage
+    CommitHash = ${Build.Vars}.CommitHash
     BuildNumber = ${Build.Vars}.BuildNumber
     ProjectName = ${Build.ProjectName}
     PSModuleManifest = ${Build.ManifestPath}
     ModulePath = $(Split-Path -Path ${Build.ManifestPath} -Parent)
 }
+if (${Build.Vars}.IsPullRequest) {
+    $BuildHelpersVariables.IsPullRequest = [bool]${Build.Vars}.IsPullRequest
```

Comment:
Here at least we're setting PowerShell variables so it can be a boolean.  Here it might make sense to set $false or $null, but then it's a slightly different behavior then when environment variables are used.